jenkins-pipelines/manager/centos-manager-backup-azure.jenkinsfile

```diff
@@ -0,0 +1,22 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    backup_bucket_backend: 'azure',
+    backend: 'aws',
+    ip_ssh_connections: 'public',
+    aws_region: '''["us-east-1", "us-west-2"]''',
+    test_name: 'mgmt_cli_test.MgmtCliTest.test_backup_feature',
+    test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
+
+    scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_mgmt_agent_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo',
+    scylla_version: '4.4',
```

Comment:
If we set it here then values we provide as parameters for job won't be applied.
Are you sure you need it here?

Reply:
There is nothing to change in this PR, cause this job definition is a copy of S3 with changed backup bucket backend. But @ShlomiBalalis can you answer this question?

Reply:
Ok, if it is not a problem for you then it is not a concern for me.